app_configs.py

```diff
@@ -41,7 +44,15 @@
 locations.
 """
 
+DEPOT_LABEL = "Depot"  # Either "Depot" or specific start location
 LOCATIONS_LABEL = "Locations"  # Either "Locations" or business specific location type
+VEHICLE_LABEL = "Vehicle"  # Singular, default "Vehicle"
+COST_LABEL = "Distance (m)"  # Either "Distance (m)" or specific distance cost description
```

Comment:
No, I considered doing that but I figured it was probably better to allow them to change it to whatever they wanted, since for Qubits I changed the whole thing & it seemed like it might come up again. 

Actually I should add a comment saying that this should include a nod to the units used.

Also open to suggestions if you have an idea of how to best approach this

Reply:
Perhaps skip the unit in the label and instead have a `SHOW_UNITS = True` option that adds whichever unit was chosen?

Reply:
I would be more for either doing unit replacing or full customization but I think the hybrid approach would be confusing. I find any time people can only change part of something they get confused.

Reply:
Ok, I'm not particularly fuzzy about this so let's keep it for now.